cob_twist_controller/src/damping_methods/damping.cpp

```diff
@@ -28,6 +28,7 @@
  ****************************************************************/
 #include "ros/ros.h"
 #include "cob_twist_controller/damping_methods/damping.h"
+using namespace Eigen;
```

Comment:
STYLE: can we keep explicit namespace ```Eigen::``` for consistency with other source files in this package!